apps/condo/domains/ticket/utils/serverSchema/TicketChange.js

```diff
@@ -0,0 +1,112 @@
+const _ = require('lodash')
+const { TicketChange } = require('./index')
+const { getById } = require('@core/keystone/schema')
+
+const createTicketChange = async (fieldsChanges, existingItem, context) => {
+    const payload = {
+        dv: 1,
+        ticket: { connect: { id: existingItem.id } },
+        ...fieldsChanges,
+    }
+    await TicketChange.create(
+        context.createContext({ skipAccessControl: true }),
+        payload,
+    )
+}
+
+const displayNameResolvers = {
```

Comment:
As for me it’s part of schema logic.  And you will not to use it as until inside somewhere.
I expect to find all of this inside the schema.

Reply:
I see following problems, when I would put `displayNameResolvers` inside a `Ticket` schema:
1. The schema will be bloated with this one-off feature
2. Implementation will be spread across this model and it will be unclear, where the feature is exactly implemented
I've thought about it at first time and it was a part of a `Ticket` module, — it was complex.